book/invalidation.md

```diff
@@ -6,13 +6,16 @@ next: skipped
 ...
 
 We [previously](animations.md) improved animation performance by
-improving our browser's graphics subsystem. But that doesn't help
+optimizing our browser's graphics subsystem. But that doesn't help
 interactions that affect layout, like text editing or JavaScript DOM
-modifications. Luckily, we can avoid redundant layout work using a
-technique called invalidation. While invalidation is traditionally
-complex and bug-prone, a principled approach and simple abstractions
-can make it managable, and the complexity is necessary to make key
-user interactions like text input much faster.
+modifications. Luckily, we can avoid redundant layout work by only
+recomputing some of it, and treating the layout tree like a form of
+cache. This *invalidaiton* technique focuses on figuring out which
```

Comment:
The technique as I hear it described is "caching and cache invalidation"; hence the
proposed changes here.